pkg/controller/installation/watch.go

```diff
@@ -0,0 +1,131 @@
+package installation
+
```

Comment:
@philbrookes if you think it would be good, I could also put the contents of `enqueueIntegreatlyOwner.go` in here too (grouping all of the stuff related to watching in this file).

Reply:
So we have removed the enqueueIntegreatlyObject now, instead using a system of @matskiv devising, that doesn't require a cluster level operator, if possible to use that, it'd be far preferred, we found that acting at cluster level caused far too many headaches.

@matskiv If I'm thinking correctly, we can delete (you did already delete?) the enqueueforintegreatly?

Reply:
I haven't deleted the enqueueforintegreatly, but we can do that, since our new Watches for CRDs are watching only specific namespaces.